client/my-sites/google-my-business/new-account/index.js

```diff
@@ -76,13 +97,23 @@ class GoogleMyBusinessNewAccount extends Component {
 
 						<div className="gmb-new-account__actions">
 							<Button
-								href={ `/google-my-business/select-location/${ siteSlug }` }
+								href={ 'https://business.google.com/create' }
+								target="_blank"
 								onClick={ this.trackCreateMyListingClick }
 								primary
 							>
-								{ translate( 'Create Your Listing' ) }
+								{ translate( 'Create Your Listing' ) } <Gridicon icon="external" />
 							</Button>
 
+							<KeyringConnectButton
+								serviceId="google_my_business"
+								forceReconnect={ true }
+								onClick={ this.trackAddListingClick }
+								onConnect={ this.handleConnect }
+							>
+								{ translate( 'Add additional Google My Business account' ) }
```

Comment:
It's not really about adding another Google My Business account, is it? My understanding is that it will allow users to start the OAuth flow over, which will give them the option to use a different Google account. As such, I would recommend `Use another Google account`.